app_server/app/util/Utility.js

```diff
@@ -60,3 +61,62 @@ exports.createLogger = function (filename) {
     ]
   });
 };
+
+var clearUserProfile =
```

Comment:
Why move these out of their respective files?
Will they be used in other files often enough to justify making them globally accessible?